src/Core/System.php

```diff
@@ -136,12 +136,13 @@ public static function jsonError($httpCode, $data, $content_type = 'application/
 	 * and adds an application/json HTTP header to the output.
 	 * After finishing the process is getting killed.
 	 *
-	 * @param mixed  $x The input content.
-	 * @param string $content_type Type of the input (Default: 'application/json').
+	 * @param mixed   $x The input content.
+	 * @param string  $content_type Type of the input (Default: 'application/json').
+	 * @param integer $options JSON options
 	 */
-	public static function jsonExit($x, $content_type = 'application/json') {
+	public static function jsonExit($x, $content_type = 'application/json', int $options = null) {
```

Comment:
According to the official function definition, the default value for `$options` should be `0`: https://www.php.net/manual/en/function.json-encode.php